Composer/packages/adaptive-form/src/components/fields/OpenObjectField/__tests__/OpenObjectField.test.tsx

```diff
@@ -72,21 +71,10 @@ describe('<OpenObjectField />', () => {
   });
 
   describe('adding more items', () => {
-    it('allows adding more items if the schema allows it', () => {
+    it('allows adding more items if the schema allows it', async () => {
       const onChange = jest.fn();
-      const { getByPlaceholderText } = renderSubject({ schema: { additionalProperties: true }, onChange });
-
-      fireEvent.change(getByPlaceholderText('Add a new key'), { target: { value: 'newKey' } });
-      fireEvent.change(getByPlaceholderText('Add a new value'), { target: { value: 'new value' } });
-
-      expect(onChange).not.toHaveBeenCalled();
-
-      fireEvent.keyDown(getByPlaceholderText('Add a new value'), { key: 'Enter' });
-      expect(onChange).toHaveBeenCalledWith(
-        expect.objectContaining({
-          newKey: 'new value',
-        })
-      );
+      const { findByText } = renderSubject({ schema: { additionalProperties: true }, onChange });
+      await findByText('Add new');
```

Comment:
Does a rejected Promise cause a unit test to fail, or do we need to wrap this in something?

Reply:
Yes, a rejected promise will cause a test to fail